Library/Formula/libsamplerate.rb

```diff
@@ -26,6 +26,19 @@ def install
     system "./configure", "--disable-dependency-tracking",
                           "--prefix=#{prefix}"
     system "make", "install"
+    rm_f "#{bin}/sndfile-resample" if build.without? "libsndfile"
+  end
+
+  def caveats
+    s = ""
+    if build.without? "libsndfile"
+      s += <<-EOS.undent
+      The example program, "sndfile-resample", if not compiled with libsndfile
```

Comment:
This is a pretty long caveat for a pretty small change. Normally caveats are for big changes in functionality or things the user needs to take action on. And the words "completely broken" are kind of scary. Could you reduce this to a single, milder-sounding line (e.g. "sndfile-resample example is disabled because this was not built with libsndfile"), or remove it entirely?